Add live app link to readme (#6)
PicnicSupermarket/hiring-experience: README.md

[![Build Status][travisci-badge]][travisci-builds]

> The source for the Picnic Hiring Experience document. You can open the Live document here at [hiring.picnic.tech][live-url].

Reviewer: ```suggestion
The source for the Picnic Hiring Experience document. View the live document at [hiring.picnic.tech][live-url].
```

(I wonder whether we can find a more descriptive label than `live-url` :thinking:.)

Author: Updated with new wording, hope it descriptive enough.

[doctoolchain-github]: https://github.com/docToolchain/docToolchain
[travisci-badge]: https://travis-ci.org/PicnicSupermarket/hiring-experience.svg?branch=master
[travisci-builds]: https://travis-ci.org/PicnicSupermarket/hiring-experience

> [live-url]: https://hiring.picnic.tech

Reviewer: Let's keep the links sorted :)

Author: Sorted 😸
